plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -193,6 +188,16 @@ class ServicesTable extends React.Component {
       }
     ];
 
+    if (!isPod) {
+      dropdownItems.splice(1, 0, {
```

Comment:
actually it's much simpler - just have to add the hidden class like we do if `isGroup || instancesCount === 0`